Tests/StackClusterTests.swift

```diff
@@ -161,7 +163,7 @@ class StackClusterTests: QuickSpec {
                     it("should be able to scroll horizontally") {
                         let vc = nestedStack()
                         vc?.loadViewIfNeeded()
-                        
+                        vc?.view.layoutIfNeeded()
```

Comment:
so why is it added to some tests which have `expect(vc).to(haveValidSnapshot())` as the first check? Sorry I may not understand you 😕